notebooks/3_KernelMethods.ipynb

```diff
@@ -95,7 +98,8 @@
     "# Changing the kernels in this cell will change them throughout the notebook tutorial.\n",
     "# The function variable designates the kernel function to use throughout the notebook.\n",
     "# The string variable is used to pass to the utility classes. \n",
-    "kernel_func = gaussian_kernel\n",
+    "kernel_params = {\"kernel\": \"precomputed\"}\n",
+    "kernel_func = rbf_kernel\n",
```

Comment:
Same comment as in #30, you can use functools to avoid having to include gamma everywhere

```
from functools import partial

kernel_func = partial(rbf_kernel, gamma=1.0)
```

Reply:
Thank you! It works.